src/en/about/guide.md

```diff
@@ -1,57 +1,95 @@
-# Lemmy Guide
+# Lemmy quick start guide
 
-Start typing...
+So you want to join Lemmy and get posting. Great! Here's how to dive straight in.
+
+## Where do I sign up?
+
+The first thing you have to do is **choose your server**. This is an extra step compared to sites like Reddit, but isn’t as difficult as it may seem.
+
+*Like with email, your identity is hosted by the server you sign up on. So for example, I joined lemmy.ml so to mention me you can type @retiolus@lemmy.ml in your post.*
+
+If what you want to talk about most fits into a clear category (maybe it’s video games or art or queer life or coding or fiction or whatever) then it might be worth making your first server one where that will primarily host that sort of content – it’ll be easier to make connections and find like-minded folk. Some consider your server to be like a neighbourhood or a venue, where the general chatter can have a specific focus.
```

Comment:
remove "or whatever", use "etc".

> then it might be worth making your first server one where that will primarily host that sort of content – it’ll be easier to make connections and find like-minded folk.

then it might be worth making a server *solely for* that sort of content - ...

> Some consider your server

Think of your server as a neighborhood ...